### PR TITLE
TRAD-92 - added account totals global header as well as other UI adju…

### PR DESCRIPTION
**Pull request by laurvasile**
…stements

### CODE REVIEW

File: src/app/core/layout/account-totals/account-totals.component.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div small-header class="small-header m-t-m" *ngIf="isETradeAccountActive">
```
Comment (sharok) [inline src/app/core/layout/account-totals/account-totals.component.html]: What is the `small-header`  attrbute? I believe it's just copy past from homer template.... I have not found such directive in the code. Am I right?

File: src/app/stocks/stock-details/stock-details.component.html [hunk 2]
```diff
@@ -103,5 +211,5 @@
         <app-stock-transactions [transactions]="scheduledTransactions"></app-stock-transactions>
       </app-panel>
     </div>
-  </div>
+  </div> -->
```
Comment (sharok) [inline src/app/stocks/stock-details/stock-details.component.html]: Don't commit commented code.... just delete it.... if you need it later you take it from VC

File: src/app/core/layout/account-totals/account-totals.component.styl [hunk 3]
```diff
@@ -0,0 +1,12 @@
+@import '../../../../styles/styles.styl'
+
+.m-b-xs
+  margin 5px 0 0 0
```
Comment (laurvasile) [inline src/app/core/layout/account-totals/account-totals.component.styl]: adjusted

File: src/app/core/layout/account-totals/account-totals.component.styl [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  font: 200 24px/1.3 Arial,sans-serif;
```
Comment (laurvasile) [inline src/app/core/layout/account-totals/account-totals.component.styl]: adjusted

File: src/app/core/layout/account-totals/account-totals.component.styl [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+small
+  font-size 100%
```
Comment (laurvasile) [inline src/app/core/layout/account-totals/account-totals.component.styl]: removed

File: src/app/core/layout/account-totals/account-totals.component.ts [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import {Observable} from 'rxjs/Observable';
+import {ToastrService} from 'ngx-toastr';
+
+@Component({
+  selector: 'account-totals',
+  templateUrl: './account-totals.component.html',
+  styleUrls: ['./account-totals.component.styl']
+})
+export class AccountTotalsComponent implements OnInit {
+  public sideBarIsOpened = true;
```
Comment (laurvasile) [inline src/app/core/layout/account-totals/account-totals.component.ts]: removed

File: src/app/core/layout/account-totals/account-totals.component.ts [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  public rightMenuIsHidden = false;
+  public totalBalance: number;
+  public isETradeAccountActive: boolean;
+
+  constructor(private router: Router,
+              private windowService: WindowService,
+              private authService: AuthService,
+              private accountService: AccountService,
+              private toastrService: ToastrService) {
+  }
```
Comment (laurvasile) [inline src/app/core/layout/account-totals/account-totals.component.ts]: removed

File: src/app/core/layout/account-totals/account-totals.component.html (hunk 1 above)
Comment (laurvasile) [inline src/app/core/layout/account-totals/account-totals.component.html]: removed

File: src/app/stocks/stock-details/stock-details.component.html (hunk 2 above)
Comment (laurvasile) [inline src/app/stocks/stock-details/stock-details.component.html]: removed

**Pull request merged**